package.json

```diff
@@ -23,7 +23,10 @@
   "scripts": {
     "analyze": "size-limit --why",
     "build": "tsdx build --format esm --target node",
-    "build-docker-ts": "tsc --project docker/tsconfig.build.json",
+    "docker-build": "docker build -t preflight .",
```

Comment:
is there a long version of this flag? (one that starts with `--`)

Reply:
Yes `--tag`
[05cb8a5](https://github.com/upleveled/preflight/pull/393/commits/05cb8a5148fa01867bd7a61f461587f0eb2eb193)